app/javascript/components/carousel/CarouselSlide.vue

```diff
@@ -30,17 +32,33 @@ export default {
   },
 
   mounted () {
-    this.setSlideStyle()
+    if (!this.isSmall() && this.isImpactsCarousel) { 
+      this.setSlideStyle(1.15)
+    } else { 
+      this.setSlideStyle()
+    }
+
     this.inputElements = getInputs(this.$el)
     this.setTabIndices()
   },
 
   watch: {
     isActive () {
       this.setTabIndices() 
+    },
+
+    currentBreakpoint () {
+      const isSmall = this.isSmall();
+      if (this.isImpactsCarousel) {
+        this.setSlideStyle(isSmall ? 1 : 1.15)
+      }
     }
   },
 
+  computed: {
+    isImpactsCarousel () { return this.$parent.$el.id == 'impacts'}
```

Comment:
I think passing it down as a boolean prop is definitely something we could do. Would you say then to remove the computed value entirely or have the computed value read the prop?

Reply:
If it comes in as a boolean through a property then you can set a default as false perhaps and always look at the property instead of a computed property. Then you can try to disentangle what it means to be an "impacts carousel" and work into the component what specific behaviour/thing has changed, and make the property represent that instead. I'm not saying this necessarily makes sense (depends on how tightly-coupled the components are generally to a specific design) but that in  principle, I would in every component I built write them agnostic of the design as much as possible, and in so-doing make it easier to re-use these components elsewhere

Reply:
I've certainly managed to remove the use of $parent, and there is now a prop which has a false default so I think it's an improvement.